tests/test_other.py

```diff
@@ -9509,6 +9509,7 @@ def test_minimal_runtime_code_size(self):
                                '-s', 'GL_SUPPORT_EXPLICIT_SWAP_CONTROL=0',
                                '-s', 'GL_POOL_TEMP_BUFFERS=0',
                                '-s', 'FAST_UNROLLED_MEMCPY_AND_MEMSET=0',
+                               '-s', 'OLDEST_SUPPORTED_CHROME_VERSION=58',
```

Comment:
If we switch to 58 as suggested earlier as the default value, this line can be removed.